### ISSUE TITLE
Replace %kernel.root_dir% to %kernel.project_dir% 

### ISSUE THREAD

**Issue by greenworld** [timiki/rpc-server-bundle#16]
As far as i see kernel.root_dir was deprecated in Symfony 4.2 https://symfony.com/blog/new-in-symfony-4-2-important-deprecations and now i'm getting errors in symfony 5.1, so had to add kernel.root_dir in services.yml parameters section manually: 
```
parameters:
    # for timiki bundle support
    kernel.root_dir: '%kernel.project_dir%/src'
```
It would be great to avoid such tricks and have this great solution for RPS server working just out of the box 

Thank you

**Comment by netesin** [comment]
@greenworld thank you for issue 👍

I`ll update package and recipes

**Comment by netesin** [comment]
Fix in 4.2 version
Add PR for update recipes https://github.com/symfony/recipes-contrib/pull/1036
